_includes/social.html

```diff
@@ -1,20 +1,19 @@
+{%- if site.email -%}<a href="mailto:{{ site.email | encode_email }}" title="email"><i class="fas fa-envelope"></i></a>{%- endif -%}
+{%- if site.orcid_id -%}<a href="https://orcid.org/{{ site.orcid_id }}" title="ORCID"><i class="ai ai-orcid"></i></a>{%- endif -%}
+{%- if site.scholar_userid -%}<a href="https://scholar.google.com/citations?user={{ site.scholar_userid }}" title="Google Scholar"><i class="ai ai-google-scholar"></i></a>{%- endif -%}
+{%- if site.publons_id -%}<a href="https://publons.com/a/{{ site.publons_id }}/" title="Publons"><i class="ai ai-publons"></i></a>{%- endif -%}
+{%- if site.research_gate_profile -%}<a href="https://www.researchgate.net/profile/{{site.research_gate_profile}}/" title="ResearchGate"><i class="ai ai-researchgate"></i></a>{%- endif -%}
+{%- if site.github_username -%}<a href="https://github.com/{{ site.github_username }}" title="GitHub"><i class="fab fa-github"></i></a>{%- endif -%}
+{%- if site.linkedin_username -%}<a href="https://www.linkedin.com/in/{{ site.linkedin_username }}" title="LinkedIn"><i class="fab fa-linkedin"></i></a>{%- endif -%}
+{%- if site.twitter_username -%}<a href="https://twitter.com/{{ site.twitter_username }}" title="Twitter"><i class="fab fa-twitter"></i></a>{%- endif -%}
+{%- if site.medium_username -%}<a href="https://medium.com/@{{ site.medium_username }}" title="Medium"><i class="fab fa-medium"></i></a>{%- endif -%}
+{%- if site.quora_username -%}<a href="https://www.quora.com/profile/{{ site.quora_username }}" title="Quora"><i class="fab fa-quora"></i></a>{%- endif -%}
+{%- if site.blogger_url -%}<a href="{{ site.blogger_url }}" title="Blogger"><i class="fab fa-blogger-b"></i></a>{%- endif -%}
+{%- if site.work_url -%}<a href="{{ site.work_url }}" title="Work"><i class="fas fa-briefcase"></i></a>{%- endif -%}
+{%- if site.wikidata_id -%}<a href="https://www.wikidata.org/wiki/{{ site.wikidata_id }}" title="Wikidata"><i class="fas fa-barcode"></i></a>{%- endif -%}
+{%- if site.strava_userid -%}<a href="https://www.strava.com/athletes/{{ site.strava_userid }}" title="Strava"><i class="fab fa-strava"></i></a>{%- endif -%}
+{%- if site.keybase_username -%}<a href="https://keybase.io/{{ site.keybase_username }}" title="Keybase"><i class="fab fa-keybase"></i></a>{%- endif -%}
+{%- if site.gitlab_username -%}<a href="https://gitlab.com/{{ site.gitlab_username }}" title="GitLab"><i class="fab fa-gitlab"></i></a>{%- endif -%}
+{%- if site.dblp_url -%}<a href="{{ site.dblp_url }}" title="DBLP"><i class="ai ai-dblp"></i></a>{%- endif -%}
+{%- if site.stackoverflow_id -%}<a href="https://stackoverflow.com/users/{{ site.stackoverflow_id }}" title="Stackoverflow"><i class="fab fa-stack-overflow"></i></a>{%- endif -%}
+{%- if site.rss_icon -%}<a href="{{ site.baseurl | prepend: site.url }}/feed.xml" title="RSS Feed"><i class="fas fa-rss-square"></i></a>{%- endif -%}
```

Comment:
same question as above: wouldn't adding `-` to all `{%` and `%}` place all these anchors on the same line in the generated code? I'd prefer placing each on a separate line, if possible.

Reply:
PR: https://github.com/alshedivat/al-folio/pull/481/commits/bd1d4f078c3ef78fb007538a35d8a81cb2771b4f

### Social in NavBar
<img width="1439" alt="Screenshot of Google Chrome (22-12-21, 14-58-45)" src="https://user-images.githubusercontent.com/28966312/147104265-df8d299f-b24e-4efc-be71-31394aab5010.png">

### Social in About (bottom)
<img width="1544" alt="Screenshot of Google Chrome (22-12-21, 15-03-39)" src="https://user-images.githubusercontent.com/28966312/147104821-bdaccbf8-bffe-4087-8e8a-02f5b33ba67b.png">